src/xenia/kernel/xam/xam_nui.cc

```diff
@@ -19,13 +19,35 @@
 #include "xenia/ui/windowed_app_context.h"
 #include "xenia/xbox.h"
 
+DEFINE_bool(NUI_Ready, false,
+            "Enable Kinect\n"
```

Comment:
I would remove this as it is misleading. I think it should be called "Enable NUI initialization", because people will extrapolate it to: "OMG KINECT"